website/src/Website.vue

```diff
@@ -86,6 +97,12 @@ export default {
 				}
 			]
 		};
+	},
+	computed: {
+		filteredTools() {
+			return this.tools.filter(tool => tool.name.toLowerCase().match(this.search.toLowerCase()));
```

Comment:
```suggestion
			return this.tools.filter(tool => {
				const matchName = tool.name.toLowerCase().match(this.search.toLowerCase());
				const matchTags = tool.tags.includes(this.search.toUpperCase());
				return matchName || matchTags;
			});
```

Reply:
so this is how you do that

Reply:
I think so, YMMV - let me know how you get on with that

Reply:
Went with having to map tool.tags as it is an array, however some of tools dont have tags so it gets an underfined error. I sorted this by wrapping `const matchTags` in an `if(tool.tags){}` but then everything broke. Decided enough time spent on error, will try again soon/ maybe we can figure it out real quick one time. 

`return this.tools.filter(tool => {
				const matchName = tool.name.toLowerCase().match(this.search.toLowerCase());
				const matchTags = tool.tags.map(s => s.toLowerCase().match(this.search.toLowerCase()));
				return matchName || matchTags;
			});`

Reply:
`map` doesn't make sense here as `map` means "run a function for each item in the array", which isn't what we are trying to do.

This PR should clarify things a lil: https://github.com/holistic-web/toolbox/pull/121

Reply:
I think I was misdirected by a stackoverflow rather than actually trying to do anything specific lol